pkgs/development/python-modules/torch/default.nix

```diff
@@ -234,13 +236,685 @@ buildPythonPackage rec {
   ];
   cudaPropagateToOutput = "cxxdev";
 
-  src = fetchFromGitHub {
-    owner = "pytorch";
-    repo = "pytorch";
-    rev = "refs/tags/v${version}";
-    fetchSubmodules = true;
-    hash = "sha256-z41VAN4l/6hyHsxNOnJORy5EQK93kSMkDHRVQrdxv7k=";
-  };
+  # the following expression has been generated using
+  # https://codeberg.org/gm6k/git-unroll
+  src =
```

Comment:
Given the size of this expression, I think it would be best to outsource it to a new file `src.nix` or similar.

Reply:
Good point. I'm now also exposing an update script through `passthru.unroll-src`.